app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -525,7 +525,7 @@ def build_toolbar_hide_button(id)
                    %w(instance_check_compliance instance_compare).include?(id)
 
     # don't hide view buttons in toolbar
-    return false if %( view_grid view_tile view_list refresh_log fetch_log common_drift
+    return false if %( view_grid view_tile view_list view_dashboard view_summary refresh_log fetch_log common_drift
```

Comment:
> what is view_summary that is added here?

@abonas Added two buttons: 
```view_dashboard``` goes to dashboard view
```view_summary``` returns to the regular summary view